Tella/Scenes/HomeView/ViewModel/FileListViewModel.swift

```diff
@@ -389,3 +389,27 @@ extension FileListViewModel {
         return FileListViewModel(appModel: MainAppModel.stub(), filterType: .all, rootFile: VaultFileDB.stub())
     }
 }
+
+extension FileListViewModel {
+    
+    func getTotalFilesInFolder(folder: VaultFileDB) -> Int {
+        let fileWalker = FileWalker(vaultDatabase: self.appModel.vaultFilesManager!.vaultDataBase)
```

Comment:
I think it's better to add this function in `VaultFilesManager`  

    func getVaultFile(vaultFilesFolders : [VaultFileDB]) -> [VaultFileDB] {
        var resultFiles : [VaultFileDB] = []
       let fileWalker = FileWalker(vaultDatabase: self.vaultDataBase)
        
        vaultFilesFolders.forEach { file in
            resultFiles.append(contentsOf: fileWalker.walk(root: file))
         }
        return resultFiles
    }



and `walk` function to get only the files in `FileWalker` class


    func walk(root:VaultFileDB) -> [VaultFileDB] {
        
        let vaultFiles = vaultDatabase?.getVaultFiles(parentId: root.id, filter: .all, sort: nil)
        
        vaultFiles?.forEach({ vaultFile in

            if vaultFile.type == .directory {
                _ = self.walk(root: vaultFile)
            } else {
                resultFiles.append(vaultFile)
            }
        })
        return resultFiles
    }